src/main/java/bio/terra/janitor/db/TrackedResource.java

```diff
@@ -0,0 +1,45 @@
+package bio.terra.janitor.db;
+
+import bio.terra.generated.model.CloudResourceUid;
+import com.google.auto.value.AutoValue;
+import java.time.Instant;
+
+/**
+ * A single tracked resource for persistence.
+ *
+ * <p>The Janitor service attempts to cleanup resources that it is tracking once they expire. This
```

Comment:
I am confused by this comments. What this tries to explain?  Does this need to be here?

Reply:
This is trying to document/explain what a tracked resource is. It's kind of fundamental to what the Janitor is doing, but we should still describe it somewhere. Having it on the value class that we're going to use everywhere to represent a "tracked resource" seemed like a good idea.
Very open to help here. Any ideas about how to make this better.

Reply:
How about adding something like 
"this class mapped to tracked_resource table in db". And what is tracked_resource table is explained in db schema?

Reply:
tweaked, ptal.